spec/controllers/admin/facilities_controller_spec.rb

```diff
@@ -0,0 +1,166 @@
+require 'rails_helper'
+
+RSpec.describe Admin::FacilitiesController, type: :controller do
+  let(:facility_id) { SecureRandom.uuid }
+  let(:facility_group) { FactoryBot.create(:facility_group) }
+  let(:valid_attributes) {
+    FactoryBot.attributes_for(
+        :facility,
+        facility_group_id: facility_group.id,
```

Comment:
```suggestion
        facility_group: facility_group,
```

Reply:
This doesn't seem to work. I'll come back to this later.